lm_eval/tasks/realtoxicityprompts/metric.py

```diff
@@ -26,6 +29,14 @@ def toxicity_perspective_api(references, predictions, **kwargs):
     url = f"https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key={API_KEY}"
 
     scores = []
```

Comment:
I think it would be good to store the toxicity scores as well.

```suggestion
    scores = []
    toxicity_scores = []
```